### PR TITLE
Translation to Korean of the tutorial Part 02 (#2794) 

### PR DESCRIPTION
**Pull request by wonderit**
feat: Translation to Korean of Part 02 - Intro to Federated Learning

This comes as a response to the initiative to translate all the
PySyft tutorials, as it is desirable to learn PySyft in our native
language.

As requested, every part of the tutorial is done via pull request
separately to:

Track better the changes in every document.

Avoid double work.

Resolves: #2794

### CODE REVIEW

**Comment by karlhigley** [comment]
@seungjaeryanlee Could you give this a look?

**Comment by seungjaeryanlee** [comment]
I will take a look later today or tomorrow!

File: examples/tutorials/translations/korean/Part 02 - Intro to Federated Learning.ipynb [hunk 1]
```diff
@@ -0,0 +1,354 @@
+{
```
Comment (seungjaeryanlee) [inline examples/tutorials/translations/korean/Part 02 - Intro to Federated Learning.ipynb]: <p>"디바이스를 이용해" 보다 "각종 도구를 이용해" 가 어떨까요?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3051/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (seungjaeryanlee) [inline examples/tutorials/translations/korean/Part 02 - Intro to Federated Learning.ipynb]: <p>"This is about as simple as models get" 은 "이것은 모델이 갖는 것처럼 아주 단순합니다" 로 번역하는 것보다는 "이보다 간단할 수 없습니다." 정도의 번역이 더 올바른 의미가 되지 않을까 싶네요.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3051/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (seungjaeryanlee) [inline examples/tutorials/translations/korean/Part 02 - Intro to Federated Learning.ipynb]: <p>"자 이제 여기 있습니다!" 가 좀 어색하게 느껴지네요. 아예 이 문장을 빼는 것도 괜찮을 것 같습니다. 또는 "완성입니다!" 정도도 괜찮겠네요.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3051/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (seungjaeryanlee) [inline examples/tutorials/translations/korean/Part 02 - Intro to Federated Learning.ipynb]: <p>"PySyft github의 Star버튼을 누르세요" 에서 "github" 는 "GitHub" 로 G 와 H를 대문자 하는것이 일반적인 것 같습니다. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3051/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Comment by wonderit** [comment]
Thanks for the feedback and sorry for the delayed fix.

**Pull request merged**